File: exercism/python/pangram/pangram.py
def is_pangram(sentence):
	alphabet = 'abcdefghijklmnopqrstuvwxyz'
	alpha_map = {}
	pangram = True
	for ch in alphabet:
		alpha_map[ch] = 0

	for ch in sentence.lower():
		if ch in alpha_map:
			alpha_map[ch] += 1
	
	for count in list(alpha_map.values()):
		if count == 0:
			pangram = False
	return pangram

File: exercism/typescript/pangram/pangram.ts
export default class Pangram {
  private sentence: string;
  constructor(sentence: string) {
    this.sentence = sentence;
  }
  alphabet = 'abcdefghijklmnopqrstuvwxyz';

  isPangram = (): boolean | string => {
    const cleanedSentence = this.sentence.replace(/[^a-z]+/gi, '').toLowerCase().split('').sort().join('');
    const lettersString = [...new Set(cleanedSentence)].join('');
    if (lettersString == this.alphabet) {
      return true;
    } else {
      return false;
    }
  }
}